include/nuttx/net/dns.h

```diff
@@ -2,7 +2,8 @@
  * include/nuttx/net/dns.h
  *
  * SPDX-License-Identifier: BSD-3-Clause
- * SPDX-FileCopyrightText: 2007-2009, 2011-2012, 2014-2015, 2018 Gregory Nutt. All rights reserved.
+ * SPDX-FileCopyrightText: 2007-2009, 2011-2012, 2014-2015, 2018
```

Comment:
This breaks spdx. Please bring back previous version (no newline).

Reply:
before there was new line added probably to fix the checkpatch but this may break spdx tag itself correct @jerpelea ? we may keep this long line to keep the spdx tag valid but @jerpelea will know for sure :-)